src/mscorlib/src/Internal/Runtime/Augments/AppContextAugments.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+
+namespace Internal.Runtime.Augments
+{
+    /// <summary>For internal use only.  Exposes runtime functionality to the Appdomain implementation in corefx.</summary>
+    public static class AppContextAugments
```

Comment:
why is the implementation of System.AppContext type also present in corefx repo ?https://github.com/dotnet/corefx/blob/master/src/System.AppContext/src/System/AppContext.cs